widgets/DynamoDB-batchsink.json

```diff
@@ -0,0 +1,98 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "DynamoDB Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
Based on the prior comments should get rid of this, if thats not possible then it should be a select widget.